draft-ietf-quic-transport.md

```diff
@@ -2703,6 +2707,8 @@ ensures that connections are not closed after new activity is initiated.
 
 To avoid excessively small idle timeout periods, endpoints MUST increase the
 idle timeout period to be at least three times the current Probe Timeout (PTO).
+This allows for multiple PTOs to expire prior to idle timeout, ensuring a single
+packet loss event cannot cause an idle timeout.
```

Comment:
That's not right, since loss events don't cause idle timeout.
```suggestion
This allows for multiple PTOs to expire during the idle timeout period, allowing
a sender to potentially recover lost packets.
```

Reply:
For idle timeout, it doesn't matter if the packets are recovered.

Reply:
I don't understand this sentence -- how can the idle timeout occur as a result of a single packet loss?

Reply:
A too-short idle timer might expire before the loss of a single packet that would otherwise have reset the timer could lead to a probe being sent.

Reply:
@Ralith : I'm having trouble parsing your sentence, but a PTO does not reset the idle timer.

Reply:
Receiving a probe does.